devel/nlohmann-json/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                      1.0
+
+PortGroup                       github 1.0
+PortGroup                       cmake 1.1
+PortGroup                       cxx11 1.1
+
+github.setup                    nlohmann json 3.2.0 v
+name                            nlohmann-json
+categories                      devel
+platforms                       darwin
+supported_archs                 noarch
+license                         MIT
+maintainers                     {gmail.com:g.litenstein @Lord-Kamina} openmaintainer
+description                     Nlohmann JSON for modern C++.
+long_description                Nlohmann is a header-only (and optionally single-header) JSON library \
+                                written in vanilla C++11. Has Unicode support\; DOM and SAX parsers, among other features.
+checksums                       rmd160  3c04b206efdeaacf49f374b33eec5b86978eef6d \
+                                sha256  2131e11d0774005e68432c5c01ff0a595e6abf0ba7979738a2017ee8a69d4001 \
+                                size    114561095
+
+configure.pre_args-append       -DBUILD_TESTING=OFF -DJSON_BuildTests=OFF
+configure.pre_args-delete       -DCMAKE_POLICY_DEFAULT_CMP0025=NEW
+configure.optflags-delete       -Os
```

Comment:
Os is kinda standard. I don't know if it does anything on a header only port, admittedly.

Reply:
Same as above, I didn't write the portfile from scratch, rather I based myself on some other Cmake port I wrote some time ago. Will remove it later.